src/shared/scrapers/US/NY/index.js

```diff
@@ -84,13 +84,16 @@ const scraper = {
   async scraper() {
     this.url = 'https://health.data.ny.gov/api/views/xdss-u53e/rows.csv?accessType=DOWNLOAD';
     this.type = 'csv';
-    const data = await fetch.csv(this.url, false);
+    const data = await fetch.csv(this.url, false, { alwaysRun: true });
```

Comment:
Don't do this. I'm guessing you were having trouble getting fresh data, and that's because you have `date` set to `false`, which caches files in the `cache` folder without a date, so you will never get new data during testing unless you delete the `cache` folder. The `cache` folder does not exist in production.

Reply:
Right, thanks @shaperilio .  Fixed in the new branch, `const data = await fetch.csv(this, this, this.url, 'default', false);`